### DISCUSSION TITLE
Will This Work With Streamlabs OBS ?

### DISCUSSION THREAD

**Discussion by dschinkel** [ExistentialAudio/BlackHole#122] [General]
I don't see this working, it doesn't give me black hole as an input device for audio with Streamlabs.  Is it possible?  I tried also a multi-output device and it still didn't show me a multi-output device in streamlabs OBS audio input settings.

**Comment by devinroth** [comment]
Send a screenshot of audio midi setup and the input device list with Streamlabs.

**Comment by quantuminformation** [comment]
OBS is the only program I haven't been able to route audio through 

![image](https://user-images.githubusercontent.com/216566/80913132-92e41300-8d42-11ea-962f-9945d92bc8bb.png)
![image](https://user-images.githubusercontent.com/216566/80913135-9bd4e480-8d42-11ea-85e2-383289bc3342.png)

**Comment by quantuminformation** [comment]
Correction, it works when its a non-agregate input

![image](https://user-images.githubusercontent.com/216566/80913160-c7f06580-8d42-11ea-83c7-713e3e98c1d5.png)

**Comment by quantuminformation** [comment]
I'm am now trying to get this to work with SL. Have you managed it?

**Comment by quantuminformation** [comment]
I managed to get it to work with SL and my aggragate setup like I use with GarageBand:


https://github.com/ExistentialAudio/BlackHole/issues/44


![image](https://user-images.githubusercontent.com/216566/94284343-84e19980-ff52-11ea-841e-3842ac708cee.png)


no aud

**Comment by devinroth** [comment]
I just released a 2ch version. Try that with OBS. I heard somewhere that OBS has issues if there is more than a certain number of channels.

**Comment by quantuminformation** [comment]
coo ill try it, Will it mean that I don't have to set up midi each time with my rode?

**Comment by devinroth** [comment]
Possibly. Let me know.
